### ISSUE TITLE
`--watch` on "node_modules" as include path is not possible

### ISSUE THREAD

**Issue by niksy** [sass/node-sass#1354]
Adding "node_modules" to `--include-path` when using `--watch` throws following error:

```sh
fs.js:488
  var r = binding.read(fd, buffer, offset, length, position);
                  ^
Error: EISDIR, illegal operation on a directory
    at Object.fs.readSync (fs.js:488:19)
    at Object.fs.readFileSync (fs.js:322:28)
    at Graph.addFile (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/sass-graph.js:61:33)
    at Graph.addFile (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/sass-graph.js:73:12)
    at /Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/sass-graph.js:47:13
    at arrayEach (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/node_modules/lodash/index.js:1289:13)
    at Function.<anonymous> (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/node_modules/lodash/index.js:3345:13)
    at baseWrapperValue (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/node_modules/lodash/index.js:2768:30)
    at LodashWrapper.wrapperValue (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/node_modules/lodash/index.js:6124:14)
    at new Graph (/Users/niksy/Sites/project/node_modules/node-sass/node_modules/sass-graph/sass-graph.js:48:8)
```

Using this without watch functionality works OK, so I suppose this is something related to sass-graph?

**Comment by niksy** [comment]
Found where the problem is.

https://github.com/sass/node-sass/blob/master/bin/node-sass#L230

sass-graph looks for "css" extension, and since I have installed `normalize.css` as dependancy, and that’s a folder, it tires to do graphing operations on directory instead of file and then it throw error. Removing CSS as extension from watcher compiles successfully.

Is there any good way to resolve this? Maybe pass extensions as options to command?
